test/server/cards/03_TWI/units/FavorableDelegate.spec.ts

```diff
@@ -0,0 +1,64 @@
+describe('Favorable Delegate', function () {
+    integration(function (contextRef) {
+        describe('Favorable Delegate\'s ability', function () {
+            describe('When played ability', function () {
```

Comment:
Jasmine will concatenate the successive test case names together, so we try to set it up so it forms a complete sentence as the convention. So for the first test case it would say `Favorable Delegate's ability When played ability should draw a card when played`, whereas we would probably want something like `Favorable Delegate's when played ability should draw a card`